tango/tango.js

```diff
@@ -0,0 +1,103 @@
+function lambda(input, callback) {
+  const { Toolbelt } = require("lp-faas-toolbelt");
+  const httpClient = Toolbelt.HTTPClient(); // For API Docs look @ https://www.npmjs.com/package/request-promise
+  const secretClient = Toolbelt.SecretClient();
+  const tokenSecretName = 'FAAS-Standard';
+  // 'FAAS-Standard'
+
+  input.payload.regionCode = "sy";
```

Comment:
Could we provide a comment here stating what the other region codes are and that it would need to be updated based on the region of the account?